### ISSUE TITLE
errors in the brightest absolute mags

### ISSUE THREAD

**Issue by mnicholl** [astrocatalogs/astrocats#16]
GRB 080319B: M = -39 -- this is the brightness of the GRB. The associated SN peaked at around -19 (http://iopscience.iop.org/article/10.1088/0004-637X/691/1/723/meta, http://iopscience.iop.org/article/10.1088/0004-637X/725/1/625/meta)

SN 2013cq: M = -29 -- also a mix of SN and GRB magnitudes

SN 2013ez: M= -29 -- also a GRB. I see a pattern emerging!

SN 2013fu: M= -28 -- GRB

"Possible supernova": M= -28 -- Redshift highly uncertain, and may be lensed

SN 2008hw: M= -26 -- GRB

SN 2004cq: M= -26 -- two conflicting redshifts, the one being used is incorrect

SN 1998ev: M= -25 -- apparent mag wrong, should be 21.3 not 14

CSS140424:123226+154840: M= -25 -- the brightest point is unrelated to the SN; it is an artefact at the edge of the CSS detector. This point is about 2 years after the actual SN. Note also that the for the 'claimed type', there was an ATel on this that would be a much better reference than CRTS or Latest Supernova pages. I don't know if this is a general trend, but ATels give a lot more information than these other sources. If possible, I would strongly encourage checking ATels for every SN and linking to the catalogue page!

SN 2010ma: M= -24 -- GRB

LSQ12axx: M= -24 -- Redshift on Latest Supernova page (0.399) is wrong, should be 0.0399 from ATel

SN 2014W: M= -24 -- Redshift on Latest Supernova page is wrong (can't work out where the number they have came from). CBET gives recession velocity rather than redshift, which I guess complicates reading things in directly from there. Corresponds to z=0.036

2001iw: M= -24 -- stumped by this one. The redshift is correct. Multiple SN cosmology papers quote the observed peak as 22 mag (you've imported this point). This looks correct for a Ia. However, the R and I band light curves peak at around 17 mag. The catalogue quotes the sources as 1,2. Source 2 is the actual paper (Barris et al), and the light curves look like they peak at 22 mag. I can't see any tabulated data. However in source 1 (Sternberg catalogue) where it seems your photometry has come from, the light curves have been shifted up by about 5 mag! I hope this isn't a common problem with light curves from this source... Or that not many of your other light curves come from there!

2002ab: M= -24 -- oh dear, looks like exactly the same issue.

2001bu: M= -24 -- took a bit of work to figure this one out! Redshift is from 'Unified SN Catalogue' (Lennarz et al). The host designation has a Q in it, which it turns out means quasar. So I think this must be a quasar rather than a SN. Might be worth going through everything from that catalogue and looking out for untyped SNe with a Q in the host name!

2001ca: M= -24 -- also from Lennarz catalogue. Not listed as a quasar, but I also see that this catalogue flags things that were never confirmed as SNe. Again, may want to filter out objects like this: one detection, never confirmed as SN, redshift also flagged as uncertain...

PS1-12qh: M= -24 -- ATel says its a QSO, not SN or LBV. Latest SN page has also got the magnitude wrong; should be 19.2, not 18.6

SN 2001iy: M= -23.7 -- see 2001iw and 2002ab

SCP06F6: M= -23.6 -- nothing wrong with the data or redshift. Maybe this isn't too important, but the absolute mag here is exaggerated simply because we need a fairly hefty K-correction at this redshift. You might want to implement M = m - DM + 2.5log(1+z) to get more realistic absolute mags for high-z SNe.

2001jp -- see 2001iw etc

2011jb -- couple of bad points in CSS light curve throwing peak mag way off

2010hu -- also a weird CSS point unrelated to SN

2001fo -- another for the weird Sternberg subclass of error

That's all I have time for for now, more soon...

**Comment by guillochon** [comment]
@mnicholl Thanks very much for the extensive error-checking!

The common denominator for _most_ of the problems is the Latest Supernova page, I have noticed a much higher incidence of errors from that page, which is understandable since David is just transcribing the ATels by hand. I'm actually probably going to change the script such that no magnitude information from that page will be imported if there is information available from any other source. For SNe for which there is no other data we'll continue to draw from that page, since David's information is correct the majority of the time.

From Sternberg I've also noticed some errors too in the past, but generally not as many. Sternberg is the largest single source by far of light curves, and most of their data is right, so I'm not too concerned. It's great though that you're finding the erroneous data, I'll check the individual events you mentioned to see if I can find a pattern in the error.

As for GRBs: Not clear to me where they fit on the supernova page in general, in principle all LGRBs have a supernova counterpart which we may or may not see, and hence are technically "supernovae" which I feel we should keep track of. But I agree the peak maximum mag should be based on the supernova light curve, not the GRB.

**Comment by guillochon** [comment]
Oh one last comment: I'd love to pull directly from ATels, but unfortunately they are written in plain English and are not easily machine-readable. This is why David Bishop is so valuable, he transcribes these for us, the downside being that he sometimes makes mistakes.

**Comment by guillochon** [comment]
@mnicholl So I think I figured out the Sternberg issue: 2001iw, 2002ab, 2001iy, 2001jp, 2001fo, etc. all come from Barris 2004: http://adsabs.harvard.edu/abs/2004ApJ...602..571B, there are 21 LCs that originate from that paper. It looks like in that paper that the light curves are presented as fluxes, not magnitudes (there is a [VizieR table](http://vizier.cfa.harvard.edu/viz-bin/VizieR-3?-source=J/ApJ/602/571/table8&-out.max=50&-out.form=HTML%20Table&-oc.form=sexa) for this data), and I'm betting that Sternberg misconverted this data into magnitudes. I'll ban the Barris data from import for now until we resolve the issue.

It would probably be good to reconvert this data to magnitudes anyway from the fluxes since there are errors reported for the fluxes, which Sternberg did not take into account when they did the conversion (none of those magnitudes have error bars). Do you have any suggestions for an accurate way of converting the flux+errors into magnitudes+errors?

**Comment by guillochon** [comment]
Some fixes here which will appear after the next import (in about a day), I linked many of the changes  so you can see how these changes were done @mnicholl:
- Magnitudes not added from Latest Supernova page if GRB is listed as an alias.
- LSQ12axx and SN2004cq's erroneous redshifts now ignored from LSNe, [correct redshift added for LSQ12axx](https://github.com/astrocatalogs/sne-internal/blob/master/LSQ12axx.json).
- SN1998ev's LSNe photometry removed, [new magnitude added](https://github.com/astrocatalogs/sne-internal/blob/master/SN1998ev.json).
- Added "QSO?" to SN2001bu, object won't be removed from list but will at least be listed as being suspicious. The "Q" appears in some host names of some legitimate supernovae, so I'm not sure if that's a good metric for removing events.
- [Added QSO type to PS1-12qh](https://github.com/astrocatalogs/sne-internal/blob/master/PS1-12qh.json), will be removed.
- Barris 2004 data drawn from ITEP [now being ignored](https://github.com/astrocatalogs/sne/commit/be819dc6810c877dc708a0d5b6d359ee6d0d4f6f), added Barris 2004 to [to do](https://github.com/astrocatalogs/sne/blob/master/TODO.md) list.

A few "won't fixes":
- For SN2001ca, I'm hesitant to remove it despite it only having one detection. It's possible data exists for the event somewhere that's not public. Policy is to retain all events with an IAU SN name unless they have a definitive typing as a non-SNe.
- CRTS data is definitely a bit flawed for some events but the CRTS data is drawn from their website which updates dynamically. In these cases I favor contacting the source of the data directly to inform them they have a pipeline issue, I've already seen with OGLE that bad points will "go away" as their pipeline is adjusted. However, one thing we could do is avoid using CRTS data if any other data is available for a given event when calculating the maximum magnitude, definitely will consider this.

And lastly a question:
- I don't have access to the CBET for SN2014W, but we support recessional velocities as inputs (they will be automatically converted to redshifts). What is the recessional velocity reported in that CBET in km/s?

**Comment by guillochon** [comment]
OK, everything except for SN2014W is resolved, attempting to get access to the CBET archive to address that one.

**Comment by guillochon** [comment]
Everything in this issue should now be resolved, closing.
